### PR TITLE
Pick up enhancements from shelbys master

### PR DESCRIPTION
**Pull request by raymondfeng**
/to @ritch

### CODE REVIEW

File: test/route-helper.test.js [hunk 1]
```diff
@@ -86,6 +86,36 @@ describe('route-helper', function() {
...
+      var result = routeHelper.routeToAPIDoc({
+        method: 'someMethod',
+        verb: 'get',
+        path: 'path',
+        description: ['1', '2', '3']
+      });
+      expect(result.operations[0].summary).to.eql("1\n2\n3");
+    });
+
+    it('should convert route.notes from array fo string to string', function () {
```
Comment (ritch) [inline test/route-helper.test.js]: typo - `fo` should be `of`

**Comment by raymondfeng** [comment]
Connect to strongloop-internal/scrum-loopback#377

**Comment by raymondfeng** [comment]
@ritch Typo fixed.

**Comment by raymondfeng** [comment]
@ritch Any more comments before I merge?

**Comment by ritch** [comment]
Nope LGTM

**Pull request merged**